docker-compose.yaml

```diff
@@ -1,4 +1,4 @@
-name: "robin-api"
+version: '3.9'
```

Comment:
nit: 
- the `version` top level element is now obsolete. (https://docs.docker.com/compose/compose-file/04-version-and-name/)
- top level `name` element helps in isolating multiple projects. Although default value is automatically inferred from the directory name, it helps to be explicit. (https://docs.docker.com/compose/project-name/)